litmus-portal/frontend/src/views/Home/ProjectInfoContainer/index.tsx

```diff
@@ -0,0 +1,113 @@
+  // const { t } = useTranslation();
+  const tabs = useActions(TabActions);
+
+  const userID = getUserId();
+
+  const projectID = getProjectID();
+  const projectRole = getProjectRole();
+
+  const [projectOwnerCount, setProjectOwnerCount] = useState<number>(0);
+  const [projectOtherCount, setProjectOtherCount] = useState<number>(0);
+  const [invitationsCount, setInvitationCount] = useState<number>(0);
+  const [projects, setProjects] = useState<Project[]>([]);
+
+  const { data: dataProject } = useQuery<Projects>(LIST_PROJECTS, {
+    onCompleted: () => {
+      if (dataProject?.listProjects) {
+        setProjects(dataProject?.listProjects);
+      }
+    },
+    fetchPolicy: 'cache-and-network',
+  });
+
+  useEffect(() => {
+    let projectOwner = 0;
+    let projectInvitation = 0;
+    let projectOther = 0;
+    projects.forEach((project) => {
+      project.members.forEach((member: Member) => {
+        if (member.user_id === userID && member.role === 'Owner') {
+          projectOwner++;
+        } else if (
+          member.user_id === userID &&
+          member.invitation === 'Pending'
+        ) {
+          projectInvitation++;
+        } else if (
+          member.user_id === userID &&
+          member.role !== 'Owner' &&
+          member.invitation === 'Accepted'
+        ) {
+          projectOther++;
+        }
+      });
+    });
+    setProjectOwnerCount(projectOwner);
+    setInvitationCount(projectInvitation);
+    setProjectOtherCount(projectOther);
+  }, [projects, dataProject]);
+
+  const projectCount = projectOwnerCount + projectOtherCount;
+
+  return (
+    <div>
+      {/* Project Level info container */}
+      <Paper className={classes.projectInfoContainer}>
+        <div className={classes.projectInfoBlock}>
+          <div className={classes.projectInfoData}>
+            <div>
+              <Typography id="projectCount">{projectCount}</Typography>
+              {projectCount === 1 ? (
+                <Typography>Project</Typography>
+              ) : (
+                <Typography>Projects</Typography>
+              )}
+            </div>
+            <div>
+              <Typography id="invitationCount">{invitationsCount}</Typography>
+              {invitationsCount === 1 ? (
+                <Typography>Invitation</Typography>
+              ) : (
+                <Typography>Invitations</Typography>
+              )}
+            </div>
+          </div>
+        </div>
+        {projectRole === 'Owner' && (
+          <ButtonOutlined
+            onClick={() => {
+              tabs.changeSettingsTabs(1);
+              history.push({
+                pathname: '/settings',
+                search: `?projectID=${projectID}&projectRole=${projectRole}`,
+              });
+            }}
+            className={classes.viewProjectButton}
+          >
+            <Typography>
+              <img src="./icons/viewProjects.svg" alt="View projects" />
+              View all projects
```

Comment:
can we have this in translation

Reply:
Translations and CSS fixes will be coming in the upcoming PR, as mentioned in PR description. cc: @vanshBhatia-A4k9

Reply:
Planning on adding a separate PR for all the translations alongside the UI fixes that are needed.